filament/backend/CMakeLists.txt

```diff
@@ -313,6 +315,7 @@ endif()
 
 if (FILAMENT_SUPPORTS_VULKAN)
     target_link_libraries(${TARGET} PUBLIC bluevk vkmemalloc vkshaders smol-v)
+    target_link_libraries(${TARGET} PRIVATE SPIRV-Tools)
```

Comment:
Oh I don't think we can do that, at least not in release. This will add tons of code to filament.

Reply:
ok, noted.  I'll revise.

Reply:
done